co-simulation tool/README.md

```diff
@@ -0,0 +1,418 @@
+
+
+# Carla-Sumo-Mosaic Co-Simulation Tool
+
+This co-simulation tool is used to run [CARLA simulator](http://carla.org/) and [SUMO simulator](https://www.eclipse.org/sumo/) with [Eclipse MOSAIC framework](https://github.com/eclipse/mosaic). The co-simulation tool can simulate both co-simulation scenarios and original Eclipse MOSAIC scenarios on both Windows systems and Linux systems.  
+
+##  **Prerequisites**
+
+For a successful running of the co-simulation tool, the following software must be installed:
+
+-  CARLA simulator
```

Comment:
Does CARLA version need to be addressed? I believe this co-simulation tool does not support all CARLA version.

Reply:
CARLA version 0.9.10 is added in the README.md.

Reply:
Is it only support CARLA 0.9.10 or it just need the CARLA start script?

Reply:
It needs CARLA start script and CARLA python API library .egg. We tested CARLA 0.9.10, 0.9.10.1, and 0.9.11. Since you asked us to use CARLA 0.9.10, the default CARLA python API library .egg is for CARLA 0.9.10 and the users should use CARLA 0.9.10..